src/app/server.rs

```diff
@@ -92,14 +87,17 @@ pub fn start_up_server(
     common: AppCommon,
     port: u16,
     address: String,
-    cache_size: usize,
     environments_regex: Regex,
     datadog: bool,
     environment_pattern: String,
     db_path: String,
     fetch_poller: u64,
     allow_fetch: bool,
+    db_max_age: usize,
 ) -> Result<()> {
+    if let Err(e) = db::clean_db(&db_path, db_max_age) {
+        warn!("Unable to clean db on startup: {:?}", e);
```

Comment:
It appears we are only cleaning the DB on startup, how often does that happen?

Reply:
Occasionally, I can add a recurring clean-up.
I'll keep the start-up clean-up to avoid a situation where performance degrades unexpectedly post-start-up with a big initial clean-up.